src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -59,13 +59,14 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     const pageCompiled = compilePage(page, contentData);
     const drupalPageDir = path.join('.', drupalUrl);
     const drupalFileName = path.join(drupalPageDir, 'index.html');
+    const layoutName = entityBundle || 'page';
 
     switch (page.entityBundle) {
       case 'health_care_local_facility':
-        addGetUpdatesFields(pageCompiled, pages);
+        // addGetUpdatesFields(pageCompiled, pages);
```

Comment:
According to the comments on addGetUpdatesFields, this is needed to add social links to these pages.

Are we sure this isn't needed?

Reply:
This was just failing our tests, wouldn't be something merged